schemas/iso19139/src/main/plugin/iso19139/layout/config-editor.xml

```diff
@@ -1220,7 +1121,615 @@
           <field xpath="/gmd:MD_Metadata/gmd:dateStamp"/>
         </section>
       </tab>
+
+      <tab id="inspire_sds"
+           mode="flat"
+           if="count(gmd:MD_Metadata/gmd:identificationInfo/srv:SV_ServiceIdentification) > 0">
+          
+          <section name="inspire_sds_cc1">
+
+              <!-- 4.4.1 TG4 : ensure gmd:DQ_DataQuality for service exists -->
+              <action type="process"
+                      process="sds-add-dataquality"
+                      if="count(gmd:MD_Metadata/gmd:dataQualityInfo/gmd:DQ_DataQuality[gmd:scope/gmd:DQ_Scope/gmd:level/gmd:MD_ScopeCode/@codeListValue='service'])=0"/>
+
+              <!-- 4.4.1 rec 1 : replace CharacterString with Anchor-->
+              <action type="process"
+                      process="sds-fix-category"
+                      if="count(gmd:MD_Metadata/gmd:dataQualityInfo/gmd:DQ_DataQuality[gmd:scope/gmd:DQ_Scope/gmd:level/gmd:MD_ScopeCode/@codeListValue='service']/gmd:report/gmd:DQ_DomainConsistency/gmd:result/gmd:DQ_ConformanceResult/gmd:specification/gmd:CI_Citation/gmd:title/gco:CharacterString)>0"/>
+
+              <!-- 4.4.1 category -->
+              <field name="sds-category"
+                     xpath="/gmd:MD_Metadata/gmd:dataQualityInfo/gmd:DQ_DataQuality[gmd:scope/gmd:DQ_Scope/gmd:level/gmd:MD_ScopeCode/@codeListValue='service']/gmd:report/gmd:DQ_DomainConsistency/gmd:result/gmd:DQ_ConformanceResult/gmd:specification/gmd:CI_Citation/gmd:title"/>
```

Comment:
By using the SDS tab it should not be possibile to add other ``gmd:DQ_DomainConsistency``.
However, we may make the layout more robust by addressing only the first ``gmd:DQ_DomainConsistency[1]`` element.